qiskit/extensions/__init__.py

```diff
@@ -51,3 +51,4 @@
 from .unitary import UnitaryGate
 from .hamiltonian_gate import HamiltonianGate
 from .simulator import Snapshot
+from .rv import RVGate
```

Comment:
Could you move this to the circuit library instead of extensions? I'm not 100% sure whether to put this in `standard_gates` or `generalized_gates` though, I'm slightly leaning towards "generalized". What do you think?

Reply:
Since the definition is not parameterized yet, and there is a test which requires standard gates to be parameterized, perhaps generalized would be better for now.